mw-server/internal/routers/favoriteUserWay.route_test.go

```diff
@@ -0,0 +1,58 @@
+package routers
+
+import (
+	"context"
+	"github.com/magiconair/properties/assert"
+	"github.com/samber/lo"
+	"mwserver/internal/auth"
+	"mwserver/internal/config"
+	"mwserver/internal/openapi"
+	"net/http"
+	"testing"
+
+	openapiGeneral "mwserver/apiAutogenerated/general"
+)
+
+func TestCreateFavoriteUserWay(t *testing.T) {
+	newConfig, err := config.LoadConfig("../../")
+	if err != nil {
+		t.Fatalf("Failed to load config: : %v", err)
+	}
+	generalApi := openapi.MakeGeneralAPIClient(&newConfig)
+	_, err = generalApi.DevAPI.DevResetDbGet(context.Background()).Execute()
+	if err != nil {
+		t.Fatalf("Failed to reset db: %v", err)
+	}
+
+	currentUserID := "7cdb041b-4574-4f7b-a500-c53e74c72e94"
+	wayUuid := "9e77b89d-57c4-4b7f-8cd4-8dfc6bcb7d1b"
+	userUuid := currentUserID
+
+	t.Run("Should create favorite user way correctly", func(t *testing.T) {
+		token, err := auth.GenerateJWT(currentUserID, newConfig.SecretSessionKey)
+		if err != nil {
+			t.Fatalf("Failed to generate JWT: %v", err)
+		}
+		request := openapiGeneral.SchemasCreateFavoriteUserWayPayload{
+			UserUuid: userUuid,
+			WayUuid:  wayUuid,
+		}
+		ctx := context.WithValue(context.Background(), auth.ContextKeyAuthorization, "Bearer "+token)
+		response, err := generalApi.FavoriteUserWayAPI.CreateFavoriteUserWay(ctx).Request(request).Execute()
+		if err != nil {
+			t.Fatalf("Failed to create favorite user way: %v", err)
+		}
+		assert.Equal(t, http.StatusNoContent, response.StatusCode)
+
+		user, response, err := generalApi.UserAPI.GetUserByUuid(ctx, userUuid).Execute()
+		if err != nil {
+			t.Fatalf("Failed to get user by uuid: %v", err)
+		}
+		favoriteUserWays := user.DefaultWayCollections.Favorite.Ways
+		uuids := lo.Map(favoriteUserWays, func(way openapiGeneral.SchemasWayPlainResponse, _ int) string {
+			return way.Uuid
+		})
+		found := lo.Contains(uuids, wayUuid)
```

Comment:
You are using assert.Contains in one place and lo.Contains in another, even though they implement the same functionality. For tests, we use the library ‘github.com/stretchr/testify/assert’, so use that one.